eb

```diff
@@ -127,6 +127,12 @@ export RSNT_EASYBUILD_MAGIC_COOKIE=263ca73bb634185aab1d1b41627fdbba
 # argument validation
 NEW_ARGS=()
 for argument in "$@"; do
+	if [[ $argument =~ ^--job-cores=[0-9]*$ ]]; then
+		kv=(${argument//=/ })
+		echo "Job-cores specified to ${kv[1]}, setting EASYBUILD_PARALLEL=${kv[1]}"
+		export EASYBUILD_PARALLEL=${kv[1]}
+		NEW_ARGS+=("$argument")
```

Comment:
This could be double, done again for ebuser below...

Reply:
I don't understand. We don't check `--job-cores` and we don't modify `EASYBUILD_PARALLEL` in the rest of the code ?

Reply:
This check is simply done for everyone, not just for when it is run as `ebuser`

Reply:
what I mean is that if run as `ebuser`, `NEW_ARGS+=("$argument")` is done twice, first time here (for everyone), and then again for `ebuser`